tests/wallet/dao_wallet/test_dao_wallets.py

```diff
@@ -204,8 +203,7 @@ async def test_dao_creation(self_hostname: str, two_wallet_nodes: OldSimulatorsA
     # Send some cats to the dao_cat lockup
     dao_cat_amt = uint64(100)
     txs = await dao_wallet_0.enter_dao_cat_voting_mode(dao_cat_amt, DEFAULT_TX_CONFIG)
-    for tx in txs:
-        await wallet_0.wallet_state_manager.add_pending_transaction(tx)
```

Comment:
there are a few places in here where you changed `wallet_0` to `dao_wallet_0`. are those referring to the same wallet or is there something subtle requiring these changes to the test?

Reply:
They don't refer to the same wallet but their `.wallet_state_manager` property is the same.  The reason I changed it is because it is important you don't actually use a wallet state manager from a different simulated environment and using the same wallet as the one that generated the transactions seemed like safer in that regard.

Reply:
(I'm going to resolve the other comments where you marked this as occuring)